modules/understanding-upgrade-channels.adoc

```diff
@@ -149,74 +100,35 @@ endif::openshift-origin[]
 * {product-version}.3
 * {product-version}.4
 
-The service recommends only upgrades that have been tested and have no serious issues.
-If your cluster is on {product-version}.1 and {product-title} suggests
-{product-version}.4, then it is safe for you to update from {product-version}.1 to {product-version}.4. Do not rely on consecutive patch numbers. In this example, {product-version}.2 is not, and never was, available in the channel. The update service will not suggest updating to a version of {product-title} that contains known vulnerabilities.
+The service recommends only upgrades that have been tested and have no serious issues. If your cluster is on {product-version}.1 and {product-title} suggests {product-version}.4, then it is safe for you to update from {product-version}.1 to {product-version}.4. Do not rely on consecutive patch numbers. In this example, {product-version}.2 is not and never was available in the channel. The update service will not suggest updating to a version of {product-title} that contains known vulnerabilities.
```

Comment:
I know this is existing text. However, when I first read the last sentence, it made me think that  {product-version}.2 contains known vulnerabilities. I read it to suggest  {product-version}.2 version was never available *because* the update service will not suggest updating to a version of {product-title} that contains known vulnerabilities.
Would it make sense to move the last sentence to the second sentence?  

_The service recommends only upgrades that have been tested and have no serious issues. The service recommends only upgrades that have been tested and have no serious issues. For example, if your cluster..._

Just a thought.

Reply:
Good suggestion about relocating that last sentence for clarity. Done.